specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerConnectivityConfiguration.json

```diff
@@ -305,6 +308,24 @@
       ],
       "description": "Properties of network manager connectivity configuration"
     },
+    "Hub": {
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "Resource Id."
+        },
+        "resourceType": {
+          "type": "string",
+          "enum": [
+            "Invalid",
+            "VirtualNetwork"
```

Comment:
To be consistent with ARM naming and values, specify resource type value as "Microsoft.Network/virtualNetworks"